adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,109 @@ endif::[]
+** credhub: 2.5.9
+** diego: 2.41.0
+** eirini: 0.0.26
+** garden-runc: 1.19.9
+** groot-btrfs: 1.0.5
+** loggregator: 105.6.3
+** loggregator-agent: 5.2.4
+** log-cache: 2.6.4
+** mapfs: 1.1.0
+** nats: 30
+** nfs-volume: 1.5.7
+** postgres-release: 26
+** pxc: 0.21.0
+** scf-helper: 1.0.13
+** sle15: 10.84
+** statsd-injector: 1.11.8
+** uaa: 74.12
+* Buildpacks:
+** binary-buildpack: 1.0.35
+** dotnetcore-buildpack: 2.3.0
+** go-buildpack: 1.9.4
+** java-buildpack: 4.27.0
+** nginx-buildpack: 1.1.2
+** nodejs-buildpack: 1.7.7
+** php-buildpack: 4.4.2
+** python-buildpack: 1.7.3
+** staticfile-buildpack: 1.5.2
+** ruby-buildpack: 1.8.3
+
+[id='sec.1_5_2.issue']
+==== Known Issues
+
+[IMPORTANT] 
+====
+* With {product} 1.5.2, the database seeder, UAA and credhub will use TLS by default to connect with an external database. Other SCF components will use the unencrypted TCP protocol. If you would like to configure the seeder, UAA & Credhub services to connect via insecure means in a trusted network, you will need to use the following parameters:
+
+`env.DB_EXTERNAL_SSL_MODE` - this is set to `true` by default and is used by the database seeder to communicate over TLS to the external database. Set it to `false` to have the seeder communicate over plain TCP.
+
+`env.UAADB_TLS` - this is set to `enabled` by default and is used by UAA to communicate over TLS to the external database. Set it to `disabled` to skip TLS for UAA connections to the external database. Please check the external database section in the documentation for other usages of this flag.
+
+`env.CREDHUB_DB_REQUIRE_TLS` - this is set to `true` by default and is used by credhub to communicate over TLS to the external database. Set it to `false` to disable TLS connections to the external database server from credhub. 
+
+Note that the external database server *must* support both TLS and non-encrypted connections; if the external database server only accepts TLS connections some SCF components will not be able to communicate with the database server.
+
+* {product} 1.5.2 has been tested successfully with external database instances hosted on Amazon RDS MariaDB and Azure MariaDB.  
+
+* The `cf-usb` service broker is not compatible with an external database hosted on Azure. We recommend using the service brokers that are provided by {azure} for OSBAPI connections to their hosted database services. 
+
+* With {product} 1.5.2 if you are using external UAA with an external database, you must set up two separate database instances; one for UAA and one for SCF. One external database instance for both an external UAA and an SCF setup is not supported and will cause data conflicts resulting in deployment failures.
+
+* If you are upgrading {product} to 1.5.2 and use an external database in 1.5.1, please contact SUSE Technical Support on guidance with the help of the backup and restore cli.
+
+Please refer to the external database section of the documentation for more configuration options.
```

Comment:
Add a link?

Reply:
Based on past practice, this gets tricky based on the various ways to contact support along with the links.